app/helpers/podcast_planner_helper.rb

```diff
@@ -5,6 +5,7 @@ module PodcastPlannerHelper
   MONTHLY_WEEKS = I18n.t([:first, :second, :third, :fourth, :fifth], scope: [:podcast_planner, :helper, :monthly_options])
   DATE_CONTROLLER = "date"
   TOGGLE_ACTION = "click->date#toggleSelect"
+  RECOUNT_ACTION = "click->count#recount"
```

Comment:
this and the `counterTarget` below are additions for the new stimulus `count_controller`. Essentially adds a counter to each date. the controller counts the ones that are selected.

Reply:
moved `counterTarget` to the input in the `_calendar` partial.